### PR TITLE
Fix some style issues

### PR DESCRIPTION
**Pull request by jkodumal**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -378,14 +377,14 @@ function evaluate(feature, user) {
     }
   }  
 
-  for (var i = 0; i < feature.variations.length; i ++) {
+  for (i = 0; i < feature.variations.length; i ++) {
```
Comment (apucacao) [inline index.js]: minor: why did you remove `var` here? technically I think that the declaration of `var i` from line 374 gets hoisted up, but it's not clear.

Comment (jkodumal) [inline index.js]: Maybe I should just hoist the declaration of `var` to the start of the function?

Comment (apucacao) [inline index.js]: yeah

**Comment by apucacao** [comment]
lgtm
